circup.py

```diff
@@ -642,7 +650,11 @@ def freeze():  # pragma: no cover
     """
     logger.info("Freeze")
     modules = find_modules()
-    if modules:
+    if JSON_OUT:
+        modules = [ (m.row[0], m.row[2]) for m in modules]
+        click.echo(json.encoder.JSONEncoder().encode(modules))
```

Comment:
Please change this (and subsequent uses of `json.encoder.JSONEncoder().encode(...)`) to the more common, shorter and simpler `json.dumps(...)`. I.e. this line becomes:

```
click.echo(json.dumps(modules))
```

Thank you..! :+1: :snake:

Reply:
I'll work this in as well, thanks